crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -522,17 +553,6 @@ impl FromWorld for BloomPipelines {
     }
 }
 
-fn extract_bloom_settings(
-    mut commands: Commands,
-    cameras: Extract<Query<(Entity, &Camera, &BloomSettings), With<Camera>>>,
-) {
-    for (entity, camera, bloom_settings) in &cameras {
-        if camera.is_active && camera.hdr {
```

Comment:
Now that you removed this, where are you checking for these conditions? Did I miss this somewhere?

Reply:
Good catch, that was my bad.
Added that check in the extract component impl for BloomSettings